core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -194,7 +194,7 @@ THE SOFTWARE.
     <div id="page-head">
       <l:pageHeader title="${%title}"
                     logoAlt="${%jenkinshead.alt}"
-                    searchPlaceholder="${%search}"
+                    searchPlaceholder="${%Search}"
```

Comment:
It will impact localizations. Would be nice to clean it up. You can also just update the english localization and keep the key in *.jelly intact

Reply:
Good spot! I've moved any changed strings to the localisation files.